README.md

```diff
@@ -1,6 +1,10 @@
+<!--BEGIN_BANNER_IMAGE--><!--END_BANNER_IMAGE-->
+
 # LiveKit Server API for JS
 
-Javascript/Typescript APIs to manage rooms and to create access tokens. This library is designed to work with [livekit-server](https://github.com/livekit/livekit-server). Use it with a Node.js backend to manage access to LiveKit.
+<!--BEGIN_DESCRIPTION-->
+This library is designed to work with [livekit-server](https://github.com/livekit/livekit-server).
```

Comment:
We probably shouldn't delete the first sentence in this change. ("Javascript/Typescript APIs to manage rooms and to create access tokens.")